package.json

```diff
@@ -105,7 +105,7 @@
     "sinon": "^15.0.0",
     "temp": "^0.x",
     "ts-node": "^10.9.1",
-    "typescript": "^5.1.3"
+    "typescript": "~5.0.4"
```

Comment:
what is the problem with higher versions of this module?

Reply:
it's incompatible with the version of typedoc we're currently stuck at